src/commandline_content.ts

```diff
@@ -47,19 +50,18 @@ async function init() {
 
 // Load the iframe immediately if we can (happens if tridactyl is reloaded or on ImageDocument)
 // Else load lazily to avoid upsetting page JS that hates foreign iframes.
-try {
-    init()
-} catch (e) {
-    // Surrender event loop with setTimeout() to page JS in case it's still doing stuff.
-    document.addEventListener("DOMContentLoaded", () => setTimeout(init, 0))
-}
 
-export function show() {
+export async function show() {
     try {
+        await init()
+        await hide()
+        await setTimeout(_=>{
```

Comment:
setTimeout doesn't return a promise, so awaiting it doesn't do what you expect.

Compare:
```js
(async () => {await setTimeout(()=>console.log(1), 100); console.log(2)})()
// 2
// {pause}
// 1

sleep = sec => new Promise(resolve => setTimeout(resolve, sec))
(async () => {await sleep(100); console.log(1); console.log(2)})()
// {pause}
// 1
// 2
```